.travis.yml

```diff
@@ -3,10 +3,18 @@ language: node_js
 node_js:
   - 'stable'
 
+before_install:
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;
+      sleep 3;
+    fi
+
```

Comment:
There are quite a few things here that I don't understand: 
1. DISPLAY=99.9. Couldn't quickly find an explanation online. 
2. The sleep command. 

I'd appreciate comments in the code for this, incase this issue pops up elsewhere in the future.

Reply:
I haven't assigned this to anyone yet for a reason...  :-)

I'm experimenting - copied this from our other repos, have no idea what it does.